src/inc/Applicability.hpp

```diff
@@ -53,8 +53,7 @@ namespace MSIX {
         void InitializeLanguages();
         void InitializeLanguages(IMsixApplicabilityLanguagesEnumerator* languagesEnumerator);
 
-        void AddPackageIfApplicable(ComPtr<IAppxPackageReader>& reader, std::string& packageName, 
-            const std::vector<Bcp47Tag>& packageLanguages, APPX_BUNDLE_PAYLOAD_PACKAGE_TYPE packageType, bool hasQualifiedResources);
+		void AddPackageIfApplicable(ComPtr<IAppxPackageReader>& reader, APPX_BUNDLE_PAYLOAD_PACKAGE_TYPE packageType, ComPtr<IAppxBundleManifestPackageInfo> bundleInfo);

```

Comment:
>	 [](start = 0, length = 1)

Tabs

Reply:
And throughout

---
In reply to: [288745380](https://github.com/Microsoft/msix-packaging/pull/146#discussion_r288745380) [](ancestors = 288745380)